src/text_page.rs

```diff
@@ -43,6 +44,18 @@ impl TextPage {
         buf.read_to_string(&mut text)?;
         Ok(text)
     }
+    pub fn stext_page_as_json(&self, scale: f32) -> Result<String, Error> {
+        let mut buf = unsafe {
+            let buf = fz_new_buffer(context(), 1024);
+            let out = fz_new_output_with_buffer(context(), buf);
+            fz_print_stext_page_as_json(context(), out, self.inner, scale);
+            fz_close_output(context(), out);
+            Buffer::from_raw(buf)
```

Comment:
this does not handle errors, it should be [wrapped in C](https://github.com/messense/mupdf-rs/blob/main/mupdf-sys/wrapper.c) to properly handle mupdf errors.